homeassistant/components/romy/vacuum.py

```diff
@@ -55,32 +53,26 @@ async def async_setup_entry(
     """Set up ROMY vacuum cleaner."""
 
     coordinator: RomyVacuumCoordinator = hass.data[DOMAIN][config_entry.entry_id]
-    async_add_entities([RomyVacuumEntity(coordinator, coordinator.romy)], True)
+    async_add_entities([RomyVacuumEntity(coordinator)], True)
```

Comment:
```suggestion
    async_add_entities([RomyVacuumEntity(coordinator)])
```
not really related, but since you're touching this line anyways. You don't need the `True`, since you're using a coordinator